client/src/components/PipelineCard.js

```diff
@@ -59,7 +59,7 @@ export const PipelineCard = ({ profileId, name, pfp, anonymous, pipeline }) => {
         condition={!anonymous}
         to={`/user/${profileId}`}
       >
-        <div className="flex flex-row justify-end items-center w-full gap-5">
+        <div className="flex flex-row justify-end items-center w-full sm:gap-5 md:gap-3 gap-0">
```

Comment:
I wouldn't have 0 gap here for mobile, since it squishes the text and pfp together, I tried out a few ones I think either gap-1 or gap-2 work. I like gap-2 better but I haven't tried it on a mobile device, just a shrunken window.

Reply:
I would also add some padding to the parent div, so the text from "Software Engineer Intern" and stuff isn't up against the edges of the page

Reply:
understood